examples/machine-learning/a3-highgpu-8g/README.md

```diff
@@ -181,6 +181,55 @@ blueprint matches the name of the user-created reservation.
   a3_maintenance_interval: ""
 ```
 
+### Using Spot VM or DWS Flex
+
+> [!IMPORTANT]
+> Select one of the provisioning models : either spot vm , dws flex or reservation
+
+In order to make use of DWS Flex Start mode with SlurmGCP, you must use the `dws_flex` variable in the `schedmd-slurm-gcp-v6-nodeset` module. 
+
+See the example below:
+
+```yaml
+  vars: 
+    a3_dws_flex_enabled: true             # enabling dws flex by setting the variable to true
+    # the rest of the variables
+
+  - id: a3_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network]
+    settings:
+      dws_flex:
+        enabled: $(vars.a3_dws_flex_enabled) 
+      # the rest of the settings, e.g. node_count_static, machine_type, additional_disks, etc.
+```
+
+To learn more about DWS Flex-Start, visit https://github.com/GoogleCloudPlatform/cluster-toolkit/blob/main/docs/slurm-dws-flex.md
+
+
+Similarly ,To make use of Spot VMs,
+In order to make use of Spot VMs with Slurm, you must use the `enable_spot_vm` variable in the `schedmd-slurm-gcp-v6-nodeset` module.
+
+See the example below:
+
+
+```yaml
+  vars: 
+    a3_enable_spot_vm: true             # enabling spot vm by settubg the variable to true
```

Comment:
leave the example to only this line change, the rest is repeated and probably not needed